use-timescale/extensions/pg-textsearch.md

```diff
@@ -0,0 +1,338 @@
+---
+title: Optimize full text search with BM25
+excerpt: Set up and optimize BM25-based full-text search using pg_textsearch extension for efficient ranked text searching
+keywords: [pg_textsearch, BM25, full-text search, text search, ranking, hybrid search]
+tags: [search, indexing, performance, BM25]
+---
+
+import EA1125 from "versionContent/_partials/_early_access_11_25.mdx";
+import IntegrationPrereqs from "versionContent/_partials/_integration-prereqs.mdx";
+
+# Optimize full text search with BM25 
+
+$PG full-text search at scale consistently hits a wall where performance degrades catastrophically. 
+$COMPANY's [pg_textsearch][pg_textsearch-repo] brings modern [BM25][bm25-wiki]-based full-text search directly into $PG,
+with a memtable architecture for efficient indexing and ranking. pg_textsearch integrates seamlessly with SQL and 
+provides better search quality and performance than the $PG built-in full-text search.
+
+BM25 scores in pg_textsearch are returned as negative values, where lower (more negative) numbers indicate better 
+matches. pg_textsearch implements the following:
+
+* **Corpus-aware ranking**: BM25 uses inverse document frequency to weight rare terms higher
+* **Term frequency saturation**: prevents documents with excessive term repetition from dominating results
+* **Length normalization**: adjusts scores based on document length relative to corpus average
+* **Relative ranking**: focuses on rank order rather than absolute score values
+
+This page shows you how to install `pg_textsearch`, configure BM25 indexes, and optimize your search capabilities using
+the following best practice: 
+
+* **Memory planning**: Size your `index_memory_limit` based on corpus vocabulary and document count
+* **Language configuration**: Choose appropriate text search configurations for your data language
+* **Hybrid search**: Combine with pgvector or pgvectorscale for applications requiring both semantic and keyword search
+* **Query optimization**: Use score thresholds to filter low-relevance results
+* **Index monitoring**: Regularly check index usage and memory consumption
+
+<EA1125 /> this preview release is designed for development and staging environments. It is not recommended for use with hypertables
+
+## Prerequisites
+
+<IntegrationPrereqs />
```

Comment:
This prereq partial needs to be updated to say "Real-time analytics", like it is in the UI